config/manager/image_config.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 images:
   system-logger: "k8ssandra/system-logger:latest"
   config-builder: "datastax/cass-config-builder:1.0-ubi7"
+  k8ssandra-client: "michaelburman290/k8ssandra-client:v0.2.0-dev.cfda014-20230724"
```

Comment:
**todo:** It would be better to get an actual release of the k8ssandra-client image with the new config builder before we merge this.

Reply:
Yep, but it wasn't available until now.